packages/mobile/src/screens/explore-screen/tabs/ArtistsTab.tsx

```diff
@@ -27,20 +27,10 @@ const useStyles = makeStyles(({ spacing }) => ({
   }
 }))
 
-// TODO: Move these somewhere (clientStore selectors)
-const getExploreUsers = (state: CommonState) => state.pages.explore.profiles
-
-const makeGetUsers = (userIds: number[]) => {
-  return (state: CommonState) => {
-    const users = state.users.entries
-    return userIds.map(id => users[id].metadata).filter(Boolean)
-  }
-}
-
 export const ArtistsTab = ({ navigation }: Props) => {
   const styles = useStyles()
-  const userIds = useSelectorWeb(getExploreUsers)
-  const profiles = useSelectorWeb(makeGetUsers(userIds))
+  const getExplore = makeGetExplore()
```

Comment:
lets move this outside the component so useSelectoWeb is always referencing the same selector function.